cognite/client/_api/files.py

```diff
@@ -560,9 +560,10 @@ def upload_bytes(
         returned_file_metadata = res.json()
         upload_url = returned_file_metadata["uploadUrl"]
         headers = {"Content-Type": file_metadata.mime_type}
-        self._http_client_with_retry.request(
+        upload_response = self._http_client_with_retry.request(
             "PUT", upload_url, data=content, timeout=self._config.file_transfer_timeout, headers=headers
         )
+        upload_response.raise_for_status()
```

Comment:
Most, if not all, errors arising from requests (or lower-level calls) are translated into Cognite-prefixed exceptions. Should this one stay as an `HTTPError` or be wrapped similarly?

Reply:
I agree. https://github.com/cognitedata/cognite-sdk-python/pull/1339/commits/eeab301747899a05af8e83a68a61c3e9c73d95e4 should cover that. 

I've added some regex processing to grab the RequestId. Would prefer to put it in a seperate method where it can be re-used (utils folder?), but I am not sure where.

Reply:
I'm not familiar enough with our lower-level API class so you need @erlendvollset to give you a definitive answer here (and on this PR in general).